courses/free-courses-es.md

```diff
@@ -93,7 +93,7 @@
 
 * [Agilidad y Lean. Gestionando los proyectos y negocios del Siglo XXI](https://miriadax.net/curso/agilidad-y-lean-gestionando-los-proyectos-y-negocios-del-s-xxi-14-a-edicion) (Miriadax)
 * [Cómo implantar grupos de mejora de procesos](https://www.edx.org/course/como-implantar-grupos-de-mejora-de-upvalenciax-gm201x-0)
-* [Gestión de proyectos software (2015)](https://ocw.unican.es/course/view.php?id=23)
+* [Gestión de proyectos software (2015)](https://ocw.unican.es/course/view.php?id=206&lang=es)
```

Comment:
Preferred
```
https://ocw.unican.es/course/view.php?id=206
```
instead of 
```
https://ocw.unican.es/course/view.php?id=206&lang=es
```

Reply:
Conflict resolved, thanks to your input!